src/patterns/payment-card-details/index.md

```diff
@@ -28,6 +30,8 @@ Let users enter payment card numbers in whatever format is familiar to them. All
 
 Present all fields on a single page, in the order in which they appear on a credit or debit card. This makes it easy for users to transcribe the information from their payment card.
 
+strong class="govuk-tag govuk-tag--grey">WCAG 2.2</strong> Make sure you leave adequate spacing between the month and year inputs. Ideally, leave at least 24px of space between the inputs, when stacked either horizontally (inline) or vertically. This will make the inputs easier to select with a mouse, touch or other input method. This is related to WCAG 2.2 success criterion [2.5.8 Target Size (minimum)](https://www.w3.org/WAI/WCAG22/Understanding/target-size-minimum.html).
```

Comment:
Fix this tag

Reply:
```suggestion
<strong class="govuk-tag govuk-tag--grey">WCAG 2.2</strong> Make sure you leave adequate spacing between the month and year inputs. Ideally, leave at least 24px of space between the inputs, when stacked either horizontally (inline) or vertically. This will make the inputs easier to select with a mouse, touch or other input method. This is related to WCAG 2.2 success criterion [2.5.8 Target Size (minimum)](https://www.w3.org/WAI/WCAG22/Understanding/target-size-minimum.html).
```